lib/fourflusher/find.rb

```diff
@@ -93,6 +93,11 @@ def initialize(device_json, os_name, os_version)
 
   # Executes `simctl` commands
   class SimControl
+    LINES_TO_REMOVE = [
+      'Install Started',
+      'Install Succeeded'
```

Comment:
I wonder if there's an `Install Failed` 🤔

Reply:
added it anyway!